MdeModulePkg/Include/Library/DeviceStateLib.h

```diff
@@ -1,4 +1,4 @@
-/** @file
+/** @file DeviceStateLib.h

```

Comment:
The integration instructions for the PR should have details for platforms to understand how to integrate and use this change (it is currently `N/A`).